CHANGELOG.rst

```diff
@@ -24,6 +24,8 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 **Added**
 
+* overheadModifier property for ``life.qbic.datamodel.dtos.business.Offer``
+
```

Comment:
Why is there the `2.3.0-SNAPSHOT` and the `2.3.0` What happened here ??

Reply:
The `2.3.0-Snapshot` section was added to the changelog during the qube force bump. I'm unsure why it's in the changelog. I can remove it and move the overheadModifier property mention to the 2.3.0 section 👍

Reply:
I think the 2.3.0-Snapshot is correct but the 2.3.0 is wrong